src/modal.js

```diff
@@ -55,15 +56,28 @@ function getRootElement() {
   return window.document.querySelector('body')
 }
 
+function buildClassName(main, additional) {
+  // No className specefied or className property is empty
+  if (!additional)
+    return main
+  // A nice and clean className, nothing special to do
+  if (typeof additional === 'string')
+    return `${main} ${additional}`
+  // react-modal will handle it
+  // see http://reactcommunity.org/react-modal/styles/classes/
+  if (typeof additional === 'object')
+    return additional
```

Comment:
Would this override `main`?

Reply:
> 

Yes it does.
Passing an object to className will override both main and ReactModal__Content classes.